src/synopsys-action/utility.ts

```diff
@@ -27,3 +27,10 @@ export async function cleanupTempDir(tempDir: string): Promise<void> {
 export function checkIfGithubHostedAndLinux(): boolean {
   return String(process.env['RUNNER_NAME']).includes('Hosted Agent') && (process.platform === 'linux' || process.platform === 'darwin')
 }
+
+export function parseToBoolean(value: string | boolean): boolean {
```

Comment:
What will happen, if we pass "truee" or any other value apart from "true"/"false"? Shall we consider that false only or we show show error/warning?

Reply:
we should consider that as false itself